include/rpp.h

```diff
@@ -146,11 +120,12 @@ extern "C" SHARED_PUBLIC rppStatus_t rppCreate(rppHandle_t* handle);
  * \retval rppStatusUnknownError
  * \retval rppStatusUnsupportedOp
  */
-extern "C" SHARED_PUBLIC rppStatus_t rppCreateWithBatchSize(rppHandle_t* handle, size_t nBatchSize, Rpp32u numThreads = 0);
+extern "C" SHARED_PUBLIC rppStatus_t rppCreate(rppHandle_t* handle, size_t nBatchSize, void* numThreadsOrStream, RppBackend backend = RppBackend::RPP_HOST_BACKEND);
 
-/*! \brief Destory RPP handle.
- * \details Function to destroy a RPP handle. To be called in the end to break down the RPP environment.
+/*! \brief Destroys RPP handle for HOST/HIP/OCL backend batch processing.
+ * \details Function to destroy a RPP handle's host/device memory allocation. To be called in the end to break down the RPP environment.
  * \param [in] handle RPP handle of type <tt> \ref rppHandle_t</tt>.
+ * \param [in] backend RPP backend to run augmentations (backend = RppBackend::RPP_HOST_BACKEND / RppBackend::RPP_HIP_BACKEND / RppBackend::RPP_OCL_BACKEND)
```

Comment:
please add description for all the parameters

Reply:
Added for both params - handle and backend